### PR TITLE
Basics cleanup

### PR DESCRIPTION
**Pull request by emmaodia**

### CODE REVIEW

**Comment by emmaodia** [comment]
GM GM @LiorAgnin Kindly review. Thanks.

File: docs/fuse-box/important-smart-contracts/major-deployed-contracts.mdx [hunk 1]
```diff
@@ -5,15 +5,30 @@ title: Major Deployed Contracts
...
+|--|---|---|
+| BlockReward | Fuse | 0x63D4efeD2e3dA070247bea3073BCaB896dFF6C9B |
 
-## Tokens&#x20;
 
-* Fuse(FUSE) - Ethereum: 0x970b9bb2c0444f5e81e9d0efb84c8ccdcdcaf84d
-* &#x20;Fuse(FUSE) - BSC: 0x5857c96DaE9cF8511B08Cb07f85753C472D36Ea3&#x20;
-* Wrapped Fuse (WFUSE) - Fuse: 0x0BE9e53fd7EDaC9F859882AfdDa116645287C629&#x20;
-* Fuse Dollar(FUSD) - Fuse: 0x249BE57637D8B013Ad64785404b24aeBaE9B098B
+| |  | |
```
Comment (LiorAgnin) [inline docs/fuse-box/important-smart-contracts/major-deployed-contracts.mdx:13]: @emmaodia Please remove this.
image

**Comment by LiorAgnin** [comment]
@emmaodia The "Working with WalletConnect" section under partners shouldn't be there. It's a guide on how smart contract wallets work with WalletConnect. It's more related to the SDK.

https://fuse-docs-git-basics-cleanup-fuse-labs.vercel.app/fuse-box/partners/walletconnect-on-fuse

**Comment by emmaodia** [comment]
> @emmaodia The "Working with WalletConnect" section under partners shouldn't be there. It's a guide on how smart contract wallets work with WalletConnect. It's more related to the SDK.
> 
> https://fuse-docs-git-basics-cleanup-fuse-labs.vercel.app/fuse-box/partners/walletconnect-on-fuse

OK. I'll add it to the DApp Development Tools section: https://fuse-docs-git-basics-cleanup-fuse-labs.vercel.app/fuse-box/deploying-smart-contracts/dapp-development-tools As one of the tools for building on Fuse. @LiorAgnin

**Comment by LiorAgnin** [comment]
> > @emmaodia The "Working with WalletConnect" section under partners shouldn't be there. It's a guide on how smart contract wallets work with WalletConnect. It's more related to the SDK.
> > https://fuse-docs-git-basics-cleanup-fuse-labs.vercel.app/fuse-box/partners/walletconnect-on-fuse
> 
> OK. I'll add it to the DApp Development Tools section: https://fuse-docs-git-basics-cleanup-fuse-labs.vercel.app/fuse-box/deploying-smart-contracts/dapp-development-tools As one of the tools for building on Fuse. @LiorAgnin

Yes, let's do that.

File: docs/fuse-box/important-smart-contracts/major-deployed-contracts.mdx (hunk 1 above)
Comment (emmaodia) [reply docs/fuse-box/important-smart-contracts/major-deployed-contracts.mdx]: Done.

**Comment by emmaodia** [comment]
Done.

**Pull request merged**